plugins/modules/vmware_guest.py

```diff
@@ -204,8 +208,13 @@
             description:
             - Enable Virtualization Based Security feature for Windows on ESXi 6.7 and later, from hardware version 14.
             - Supported Guest OS are Windows 10 64 bit, Windows Server 2016, Windows Server 2019 and later.
-            - The firmware of virtual machine must be EFI.
+            - The firmware of virtual machine must be EFI and secure boot must be enabled.
+            - Virtualization Based Security depends on nested virtualization and Intel Virtualization Technology for Directed I/O.
             - Deploy on unsupported ESXi, hardware version or firmware may lead to failure or deployed VM with unexpected configurations.
+        vvtd:
```

Comment:
"IOMMU" seems more generic for Intel and AMD, and it's also displayed "I/O MMU" on vCenter UI.

Reply:
You're right. I just took vvtd because that's how it's called in the API. I'll rename the parameter.